_posts/blog/2016-12-22-dwh-part1_getting_started.md

```diff
@@ -0,0 +1,134 @@
+state-of-the-art techniques and best practices of building a DWH, the amount of possible products and solutions was overwhelming. Therefore, we decided to engage a consultant
+with domain expertise to guide us the first steps.
+
+## Basic components
+
+In collaboration with the consultant, we identified basic components of a DWH for our use case: _sources_, _staging area_, _DWH core_, and _data mart_. The points and diagram below
+describe these components:
+
+1. Sources: We already introduced three kinds of data in the first section but we did not mention the data sources. The content metadata, in agreement with the content publisher,
+   is usually directly extracted from the publisher's source and stored into Elasticsearch. Data about user preferences is provided by our own in-app service and stored into a
+   PostgreSQL database. User behavior data is both collected by our own in-app service and by a third party service. User behavior data is then stored into an S3 bucket.
+2. Staging area_: The staging area is the first destination of all the data. It contains data which was extracted from the sources and might be incomplete and unadjusted. This
+   location can be used for data cleansing and transformation.
+3. DWH core: It is planned to load the cleaned and transformed data into the DWH core. The DWH core is intended to contain the entire data for general purpose analysis. This
+   contrasts the previous definitions which describe a DWH as subject-oriented. We wanted to keep that door open because analysis purposes were not clearly defined at that stage
+   and keeping the platform flexible was important to support comprehensive analysis by data scientists and data mining experts.
+4. Data mart: For our use case, the data mart is the business layer which is subject-oriented and optimized for analysis purposes. The data is loaded from the DWH core via a
+   separated ETL process and contains aggregated information which can be queried by business analysts.
+
+<center>
+<picture>
+  	<a href="/images/blog/dwh_journey/dwh_components.png"><img src="/images/blog/dwh_journey/dwh_components.png" alt="DWH components" width="850"></a>
+   	<figcaption>Figure 1: DWH components</figcaption>
+</picture>
+</center>
+
+
+## Processing steps and tools
+
+From the consultancy, we learned that a DWH can be implemented as a flexible process chain which is applied to collect data from different sources. The collected data is then
+stored in a central location and used to generate reports, create insights, and support business decision-making. We identified the following process steps: _extract, transform
+and load (ETL)_, _scheduling_, _monitoring_, _storage_, and _reporting_. As briefly described below, each of these processing steps can be covered by a variety of tools:
+
+* The ETL process is one of the most important processing steps of the DWH. It is applied to extract, transform, and load data from various sources
```

Comment:
can you also bold words like "ETL process", "Scheduling" , "Monitoring", "Sorage" and "reporting" ?